proto/telemetry.proto

```diff
@@ -19,6 +19,16 @@ enum TelemetryNodeType {
   TELEMETRY_NODE_TYPE_COMPACTOR = 4;
 }
 
+enum TelemetryClusterType {
+  TELEMETRY_CLUSTER_TYPE_UNSPECIFIED = 0;
+  TELEMETRY_CLUSTER_TYPE_SINGLE_NODE = 1;
+  TELEMETRY_CLUSTER_TYPE_TEST = 2;
+  TELEMETRY_CLUSTER_TYPE_DOCKER_COMPOSE = 3;
+  TELEMETRY_CLUSTER_TYPE_KUBERNETES = 4;
+  TELEMETRY_CLUSTER_TYPE_HOSTED = 5;
```

Comment:
What is `HOSTED` ?

Reply:
It means the cluster is on Risingwave Cloud.

Reply:
I see. Perhaps you mean `CLOUD_HOSTED`